app/marathons/page.tsx

```diff
@@ -0,0 +1,140 @@
+"use client";
+
+import {
+	Container,
+	Row,
+	Col,
+	Button,
+	Card,
+	Accordion,
+	ListGroup,
+} from "react-bootstrap";
+import externalSourceData from "../../public/info/external-sources.json";
+import { EXTERNAL_SOURCE_KEYS } from "../consts/external-source-keys.const";
+
+export default function Page() {
+	const marathonExternal = externalSourceData.find(
+		(external) => external.key == EXTERNAL_SOURCE_KEYS.MARATHON_FORM
+	);
+	return (
+		<Container className="my-3 my-xl-5">
+			<Row>
+				<Col xl={{ span: 8, offset: 2 }} className="my-2 img-overlay">
+					<h1 className="text-center">Zapisy na bieg trwają!</h1>
+					<p className="text-center px-5 px-md-0">
+						Biegnij dla zdrowia i zabawy! Dołącz do tysięcy biegaczy
```

Comment:
When making responsive UI design it's important to remember polish grammar rules. [As a general rule, you should not leave single-letter conjunctions and prepositions at the end of a line](https://sjp.pwn.pl/poradnia/haslo/Sieroty-na-koncu-wiersza;18684.html#:~:text=W%20wierszach%20tekstu%2C%20kt%C3%B3re%20licz%C4%85%20powy%C5%BCej%2040%20znak%C3%B3w%2C,dopuszczalne%20tylko%20przy%20bardzo%20w%C4%85skim%20%C5%82amie%20%28np.%20gazetowym%29.)

You can achievie this by using non-breaking space `&nbsp;`

```suggestion
						Biegnij dla zdrowia i&nbsp;zabawy! Dołącz do tysięcy biegaczy
```